synfig-studio/src/gui/instance.cpp

```diff
@@ -1218,7 +1218,9 @@ Instance::make_param_menu(Gtk::Menu *menu,synfig::Canvas::Handle canvas, synfiga
 			param_list.add("selected_value_desc",selected_value_desc);
 	}
 
-	param_list.add("child",canvas_view->get_work_area()->get_active_bone_value_node());
+	if (std::string("bone") == canvas_view->get_smach().get_state_name()) {
```

Comment:
I don't think you need to explictly cast here.

Reply:
`get_state_name` returns `const char *` so to compare it, you need to cast. Or use something like `strcmp` which I don't want to use.

Reply:
Oh, ok. I thought it would return std::string. Sorry.